src/main/webapp/WEB-INF/jsp/help.jsp

```diff
@@ -0,0 +1,193 @@
+                keep this in mind if information in the "Grades" and "Assignments" tabs in Canvas is not completely
+                up to date.</p>
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="headingThree">
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThree"
+                   aria-expanded="false" aria-controls="collapseThree">
+                    Editing and Deleting Assignments
+                </a>
+            </h4>
+        </div>
+        <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
+            <div class="panel-body">
+                <p><strong>DO NOT EDIT THE ATTENDANCE ASSIGNMENT FROM THE ASSIGNMENTS TAB IN CANVAS.</strong> </p>
+                <br>
+                <h4>Editing: </h4>
+                <p>If you'd like to change the assignment name, the total points, or the weight of any particular
+                attendance status, make all of your changes through the K-State Attendance application. Make the desired
+                changes by creating a new Setup on the Course Setup page then saving the setup for the appropriate
+                section.</p>
+                <br>
+                <h4>Deleting: </h4>
+                <p>Deleting an attendance assignment is done through the Assignments tab in Canvas just like any other
+                assignment. However, if you'd like to delete all of the attendance assignments you've created, simply
+                uncheck the "Convert Attendance to Assignment" checkbox. The dialog pictured below will pop up and answering
+                "Yes" will remove all saved Setups and all attendance assignments. This will <strong>NOT</strong> have
+                any affect on the attendance records you have saved in the course.</p>
+                <br>
+                <p><strong>NOTE:</strong> If you receive an error after recently editing or deleting an assignment, please send us an email including the
+                URL of the error page in the body of the email.</p>
+
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="headingFour">
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+                    Understanding <em>Attendance Weights</em>
+                </a>
+            </h4>
+        </div>
+        <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
+            <div class="panel-body">
+                <img src="${context}/img/WeightsExample.png">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)